sysutils/apl_select/files/base

```diff
@@ -0,0 +1 @@
+bin/apl
```

Comment:
What is the function of this file?

Reply:
Note that this file lacks a terminating carriage return...

Reply:
Added.